FE/src/components/common/QuestionSelectionBox/QuestionAddForm.tsx

```diff
@@ -1,14 +1,18 @@
 import useInput from '@hooks/useInput';
-import useQuestionAdd from '@hooks/useQuestionAdd';
+import useQuestionAddMutation from '@hooks/useQuestionAdd';
 import { css } from '@emotion/react';
 import { Button, InputArea } from '@foundation/index';
 
 type QuestionAddFormProps = {
   categoryId: number;
 };
 
+/**
+ * 현재 사용하지 않는 컴포넌트 입니다.
+ * 나만의 질문에서 사용했던 질문을 바로 추가하는 컴포넌트로 사용하고 있습니다.
+ */
```

Comment:
[p-2]
```suggestion
/**
 * @deprecated
 */
```

이렇게 jsDocs로 남겨두면 아래와 같이 더 명확하게 안내할 수 있을 것 같아요!
<img width="675" alt="image" src="https://github.com/boostcampwm2023/web14-gomterview/assets/57657868/07999d74-e7b4-48a1-9861-e7e283ab43b0">

Reply:
https://github.com/boostcampwm2023/web14-gomterview/pull/119/commits/513bd7a2a8828e64d08fcf0fc4ad920fce960f72

반영 완료